### PR TITLE
NEXT-37105 - new payment handling

### PR DESCRIPTION
**Pull request by lernhart**

### CODE REVIEW

File: guides/plugins/plugins/checkout/payment/add-payment-plugin.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ nav:
 
 # Add Payment Plugin
 
+::: warning
+With Shopware 6.6.5.0, the payment handling was refactored.
+Most of this documentation is deprecated and obsolete with 6.7.0.0.
+
+The new payment handling is done via a single `AbstractPaymentHandler`.
```
Comment (Isengo1989) [inline guides/plugins/plugins/checkout/payment/add-payment-plugin.md:14]: @lernhart the current branch next-6.7 is in progress at the moment and typically will only be merged when the release is out. Was there a different approach discussed with @bojanrajh ? Or is all of it already available in 6.6.5.0 ?

Comment (lernhart) [reply guides/plugins/plugins/checkout/payment/add-payment-plugin.md:14]: It is available with 6.6.5.0 and deprecated. I discussed this with the Docs team. We should add a 6.7 (upcoming) version to the selector in the Docs, so developers can see the upcoming changes. 

If you have a look at other documentations, symfony for example, you will see that they have a similar approach.

**Review by cyl3x — APPROVED** [review]

**Comment by Isengo1989** [comment]
@bojanrajh fyi - I just tagged it 6.7 to merge when that branch goes live. Destination is v6.6

File: guides/plugins/plugins/checkout/payment/add-payment-plugin.md (hunk 1 above)
Comment (bojanrajh) [reply guides/plugins/plugins/checkout/payment/add-payment-plugin.md:14]: Confirmed, the `v6.7 (unreleased)` version will be added as soon as [this PR](https://github.com/shopware/docs/pull/1423) will have some changes.

**Pull request merged**